agent-codeless/src/main/resources/rp-logger-config/diagnostics.appender.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<included>
+    <property scope="local" name="ai.appsvc.logfile.name" value="${ai.logfile.name:-application-insights-extension}" />
+    <property scope="local" name="ai.appsvc.logfile.ext" value="${ai.logfile.ext:-.log}" />
+    <property scope="local" name="ai.appsvc.logfile.old.ext" value="${ai.appsvc.logfile.ext:-.log}.old" />
```

Comment:
```suggestion
    <property scope="local" name="ai.appsvc.logfile.name" value="application-insights-extension" />
    <property scope="local" name="ai.appsvc.logfile.ext" value=".log" />
    <property scope="local" name="ai.appsvc.logfile.old.ext" value=".log.old" />
```
This seems less confusing to me, compared to referencing properties defined in the other file.